content/docs/command-reference/commit.md

```diff
@@ -140,83 +138,83 @@ stage with `dvc run --no-commit`, or reproduce an entire pipeline using
 development of the stage is finished, `dvc commit` can be used to store data
 files in the cache.
 
-In the `featurize.dvc` stage, `src/featurize.py` is executed. A useful change to
-make is adjusting a parameter to `CountVectorizer` in that script. Namely,
-adjusting the `max_features` value in the line below changes the resulting
-model:
+In the `featurize` stage, `src/featurization.py` is executed. A useful change to
+make is adjusting the `max_features` parameter to `CountVectorizer` in that
+script. The parameters are defined in `params.yaml` file. Updating the value of
+`max_features` to 6000 in `params.yaml` changes the resulting model:
 
-```python
-bag_of_words = CountVectorizer(stop_words='english',
-            max_features=6000, ngram_range=(1, 2))
+```
+featurize:
+  max_features: 6000
+  ngrams: 2
 ```
 
-This edit introduces a change that would cause the `featurize.dvc`, `train.dvc`
-and `evaluate.dvc` stages to execute if we ran `dvc repro`. But if we want to
-try several values for `max_features` and save only the best result to the
-cache, we can run it like this:
+This edit introduces a change that would cause the `featurize`, `train` and
+`evaluate` stages to execute if we ran `dvc repro`. But if we want to try
+several values for `max_features` and save only the best result to the cache, we
+can run it like this:
 
 ```dvc
-$ dvc repro --no-commit evaluate.dvc
+$ dvc repro --no-commit evaluate
 ```
 
-We can run this command as many times as we like, editing `featurize.py` any way
-we like, and so long as we use `--no-commit`, the data does not get saved to the
-cache. Let's verify that's the case:
+We can run this command as many times as we like, editing `featurization.py` any
```

Comment:
editing `params.yaml` ?

Reply:
Yeah fixed this here and in the next instance. The changes are affecting `featurization.py` but the `params.yaml` is being edited actually